src/components/FooterMobile.vue

```diff
@@ -20,20 +19,15 @@
 
 <script>
 import { mapState } from 'vuex';
-import Accounts from '../pages/Accounts.vue';
 import QuickId from './QuickId.vue';
 import ApproveMessage from './ApproveMessage.vue';
 import ApproveTransaction from './ApproveTransaction.vue';
 
 export default {
   components: {
-    QuickId, Accounts, ApproveMessage, ApproveTransaction,
-  },
-  props: {
-    showBackButton: { type: Boolean, default: false },
+    QuickId, ApproveMessage, ApproveTransaction,
```

Comment:
@davidyuk 

Isn't it better if it is like:

```
{
  QuickId, 
  ApproveMessage, 
  ApproveTransaction,
}
```

Reply:
I'm not sure, this style is allowed by current linter settings. If a developer adds additional components into this line, linter will ask to write as you proposed, but now it is not necessary.